src/alert/alert.style.tsx

```diff
@@ -143,3 +147,31 @@ export const TextContainer = styled.div`
     flex: 1;
     overflow-wrap: anywhere;
 `;
+/*
+ * Height is calculated based on text line height
+ * max-height: text-line-height * maximum number of lines
+ */
+
+export const TextWrapperContainer = styled.div<TextContentProps>`
+    ${(props) => {
+        const gradient =
+            "linear-gradient(to bottom, black 50%, transparent 100%)";
+        if (!props.$showMore && props.$maxHeight)
+            return `
+                height: ${props.$maxHeight}px;
```

Comment:
actually i thought the max height would mean the entire component's max height rather than the text content?